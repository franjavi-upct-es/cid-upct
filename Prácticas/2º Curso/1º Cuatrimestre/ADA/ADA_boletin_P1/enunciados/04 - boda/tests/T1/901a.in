50
90 11
9 17 3 10 13 14 1 27 10 3
10 6 10 24 15 13 13 6 23 10 5
14 10 8 18 3 16 8 21 12 28 5 28 30 6 10
8 29 29 15 1 2 20 10 26
9 15 30 2 7 9 28 29 10 28
12 12 13 23 24 24 13 29 13 12 26 23 17
10 21 2 25 22 13 26 9 9 10 9
14 17 17 22 7 19 19 22 30 23 7 24 8 11 14
14 22 9 4 1 25 24 24 11 8 6 6 8 7 8
12 8 16 25 29 22 13 17 5 4 9 3 27
8 13 2 28 27 3 1 27 19
85 12
13 16 9 26 23 15 3 9 15 10 25 13 1 29
14 28 2 22 30 21 30 13 22 27 1 16 19 27 4
5 30 24 20 8 20
13 15 14 13 29 24 29 3 16 27 24 13 28 8
9 18 7 17 2 25 17 9 13 5
11 18 26 6 29 3 17 11 9 1 15 29
12 13 2 9 9 17 22 7 24 26 16 22 4
5 8 12 18 13 16
14 22 11 27 20 5 14 23 14 6 7 4 29 20 5
5 20 14 12 18 29
9 4 13 2 12 20 13 29 24 20
10 15 22 9 27 27 22 19 2 19 17
68 7
2 2 7
2 15 11
3 5 9 30
3 3 4 29
11 24 22 27 6 29 10 14 25 28 27 5
13 15 21 26 23 19 19 29 28 25 1 24 29 10
4 8 4 19 6
81 14
13 28 9 17 8 22 3 5 10 7 26 24 20 13
2 30 23
1 27
9 7 20 7 8 5 6 11 15 11
13 19 22 10 27 8 9 11 3 14 20 9 1 6
6 5 13 19 27 19 7
1 18
9 11 17 23 9 28 8 19 10 26
12 11 14 10 20 24 12 25 6 12 25 11 24
7 23 12 25 4 19 29 13
11 1 29 29 9 18 28 20 19 23 21 30
11 30 19 14 3 5 19 7 29 21 30 19
2 11 13
3 21 3 21
78 5
13 18 26 29 15 7 17 30 27 8 28 19 18 11
1 22
8 27 20 4 18 8 9 29 12
3 11 6 29
14 23 10 25 18 8 1 16 16 30 12 24 19 30 11
40 3
13 13 1 14 16 19 14 25 9 17 3 19 23 2
2 7 11
10 16 10 5 1 25 26 12 10 15 4
77 16
2 15 10
12 20 17 10 3 3 18 12 6 29 26 29 8
2 9 4
9 10 30 9 26 25 20 6 1 15
4 6 14 24 7
12 13 24 23 8 26 3 19 23 23 6 21 30
11 21 25 15 22 24 23 18 19 4 23 19
11 2 17 24 25 23 7 7 8 22 6 26
2 24 10
12 29 1 29 27 13 24 3 5 17 25 22 5
13 6 16 1 7 2 24 23 16 23 21 16 6 27
13 21 12 12 28 3 12 26 29 25 19 24 29 28
6 12 2 8 9 9 30
9 2 16 29 18 30 19 25 5 15
13 25 19 8 14 21 20 1 11 6 12 4 26 9
6 29 2 21 7 11 13
50 12
12 11 21 27 21 15 23 6 11 9 16 11 22
12 22 14 16 27 25 20 14 25 3 12 27 15
4 29 27 22 2
12 2 23 12 22 29 4 19 2 12 4 12 25
3 3 1 17
9 25 28 26 12 30 7 30 6 16
14 3 29 29 18 30 13 22 13 4 25 24 5 29 19
12 15 20 10 15 28 22 2 25 17 13 16 15
14 21 22 1 15 20 21 25 11 26 16 15 29 2 8
11 22 19 10 7 8 11 13 27 2 6 21
6 10 28 24 13 18 15
3 25 26 26
50 6
1 4
12 11 27 20 13 19 8 14 17 15 24 21 11
7 27 23 26 6 20 19 11
9 25 15 23 20 2 11 17 14 6
6 24 3 25 28 13 24
9 29 30 26 19 10 12 15 24 29
70 15
11 23 4 25 29 18 14 1 21 22 6 26
6 30 20 6 19 2 30
12 30 29 9 11 1 21 17 24 11 29 21 20
9 24 14 12 11 20 4 23 11 9
5 22 30 30 28 19
12 27 2 23 25 10 25 17 22 12 3 3 10
1 22
5 16 27 18 18 16
7 3 19 21 13 10 21 12
7 1 5 3 2 20 20 3
12 28 25 17 30 27 19 22 10 3 29 6 20
1 22
14 11 2 22 23 11 4 26 18 26 1 12 27 12 23
2 25 21
4 4 20 4 22
53 13
10 2 10 13 10 23 6 12 24 27 4
13 23 30 14 18 22 17 15 3 2 28 19 22 13
14 3 8 13 6 20 28 30 21 2 1 14 7 13 7
6 8 3 26 29 16 13
6 25 19 14 26 8 25
3 12 8 11
11 21 9 30 10 30 21 12 30 4 18 4
14 13 12 5 30 2 12 5 14 6 15 19 23 23 5
5 4 13 4 23 25
10 22 4 3 12 15 24 7 25 28 9
5 1 13 7 2 16
11 7 14 17 18 6 1 22 29 5 26 2
1 20
82 12
14 21 14 22 14 12 16 11 20 15 3 24 13 5 2
9 3 15 1 20 20 2 4 18 28
13 11 16 11 7 17 18 19 22 9 3 26 17 5
13 1 8 23 13 4 24 19 6 8 19 18 19 12
11 29 1 3 9 8 13 8 25 22 26 8
9 20 3 9 25 10 1 2 2 5
11 17 23 2 24 3 19 12 7 1 2 7
4 3 15 30 10
5 22 27 8 14 9
13 14 3 4 6 4 5 10 30 21 2 1 6 4
6 9 10 4 11 9 29
11 23 29 14 23 20 2 22 25 10 16 30
73 19
3 7 15 6
2 5 7
5 2 2 10 11 4
9 13 12 4 25 26 2 30 10 13
3 23 29 3
13 20 6 27 24 5 12 29 2 8 27 30 2 20
11 12 23 6 16 26 9 2 13 2 1 14
4 23 7 26 25
11 15 23 3 30 27 6 20 28 6 16 28
3 5 20 10
4 25 25 7 25
8 20 18 18 25 23 10 1 18
9 29 24 19 2 23 15 29 12 12
4 27 1 2 24
13 11 5 7 27 11 2 14 22 11 1 17 4 3
9 13 29 8 7 9 1 29 23 21
1 26
11 29 19 26 14 1 29 18 29 17 21 22
7 4 3 23 12 6 25 13
71 5
2 16 23
4 15 7 2 17
1 18
13 13 6 21 13 4 30 11 12 12 3 12 16 5
7 19 2 21 1 11 6 12
44 18
3 11 5 17
7 29 4 25 11 9 15 15
4 7 25 16 18
1 19
13 23 15 14 24 6 14 5 3 26 1 22 10 3
12 26 13 16 22 7 18 30 14 2 26 20 27
9 29 15 21 24 30 28 29 23 3
6 19 5 21 11 26 1
13 14 18 17 21 9 16 9 1 29 10 18 10 6
10 8 13 2 23 12 29 21 26 23 17
5 19 7 17 7 7
4 20 17 22 10
3 7 10 9
13 20 26 14 25 7 14 29 1 6 10 21 19 28
4 5 4 2 11
8 8 10 25 19 18 17 21 4
3 30 13 20
11 8 25 6 7 8 4 7 14 6 27 24
43 12
2 6 26
8 9 3 9 4 14 18 12 4
12 26 25 4 7 6 3 2 11 1 9 6 7
4 3 26 7 5
13 26 2 22 26 11 17 26 6 22 14 9 25 5
4 11 30 3 16
12 4 18 25 4 24 23 18 26 10 16 23 29
10 16 20 7 26 28 24 23 19 7 1
9 3 27 8 24 29 23 10 24 11
12 27 26 18 6 21 20 22 5 18 25 21 29
11 8 27 16 1 7 15 1 4 9 19 11
11 17 3 12 2 5 7 29 30 16 26 13
54 7
5 14 11 7 13 3
5 1 11 7 7 25
1 2
7 18 12 27 26 7 8 20
3 6 18 3
8 13 15 10 22 1 24 24 30
2 19 6
98 9
4 26 23 2 28
9 11 1 14 29 7 21 18 10 19
1 12
8 1 26 25 22 18 10 8 17
13 26 14 26 24 17 22 16 19 19 2 21 19 15
1 18
4 28 27 8 24
10 23 25 25 17 8 12 27 15 29 25
4 12 21 25 29
93 12
11 22 4 21 2 10 10 11 29 7 7 28
1 6
12 17 30 29 16 12 25 1 10 11 2 13 23
5 3 26 28 3 9
11 24 11 25 25 21 23 1 20 21 23 25
12 1 25 30 17 6 16 9 7 27 10 11 19
8 6 7 18 8 15 10 23 17
12 17 30 19 10 19 1 2 5 24 25 29 24
13 26 9 19 24 27 20 27 8 10 2 14 27 1
10 28 24 7 24 10 6 4 19 24 26
5 20 19 7 11 12
13 28 13 19 22 9 30 10 8 9 11 21 27 3
41 15
12 17 10 6 22 13 16 7 30 28 26 18 26
6 22 4 26 4 14 9
10 14 18 12 14 20 24 11 22 4 5
4 20 6 15 3
4 22 1 9 20
1 26
1 16
9 3 11 4 16 19 30 21 6 11
13 17 4 15 9 30 11 18 11 16 24 14 25 15
12 3 26 24 20 3 10 21 27 12 24 12 23
11 25 28 25 29 7 21 5 15 20 7 2
2 22 25
5 8 1 11 2 27
12 13 29 27 3 25 9 19 28 1 3 14 20
8 4 26 18 8 2 29 6 25
82 9
3 25 26 12
13 19 8 22 2 28 11 26 14 19 6 3 11 9
11 22 28 20 10 15 20 3 5 17 28 24
12 8 10 23 11 6 12 18 20 5 7 30 22
2 10 28
3 12 28 1
11 25 12 4 1 1 29 5 10 26 28 15
9 8 7 6 5 10 15 24 6 22
7 20 11 24 17 25 5 14
57 20
2 7 25
12 29 23 4 8 10 23 14 5 22 21 2 27
4 16 12 20 7
7 9 18 12 17 4 17 22
9 8 21 5 24 20 25 17 15 25
10 29 8 22 21 20 23 9 12 1 12
7 7 8 1 16 12 17 20
8 30 18 27 20 14 20 1 30
4 15 24 16 5
10 29 17 21 14 25 24 14 29 16 20
4 8 28 9 16
1 28
5 4 24 4 9 6
2 8 25
12 2 2 7 25 1 23 15 6 10 9 19 8
14 30 5 24 27 14 9 5 11 24 8 27 19 8 24
11 8 18 16 9 20 14 3 12 7 9 9
12 9 19 15 25 18 19 18 7 2 18 11 5
3 11 1 22
2 24 28
49 3
11 25 14 11 19 17 17 20 25 24 28 13
4 22 1 18 1
9 12 11 9 16 13 11 8 26 21
57 3
1 25
4 2 1 30 12
1 8
68 12
3 21 24 22
8 16 9 5 14 12 15 15 19
9 17 26 15 7 10 29 14 27 10
3 27 9 10
1 8
13 24 30 23 10 13 26 25 22 30 1 25 6 7
10 25 23 1 1 22 11 21 27 7 1
9 3 1 12 15 9 6 1 30 28
8 13 24 26 26 15 26 20 21
10 3 7 17 4 29 30 14 19 26 12
5 28 14 20 1 20
4 7 12 27 26
53 1
13 17 26 4 4 16 16 28 18 22 6 13 20 5
97 10
10 7 19 19 12 8 19 24 27 17 5
9 13 18 16 23 26 11 26 21 18
13 18 6 24 23 18 5 27 6 5 18 4 15 6
6 22 17 23 10 3 27
6 7 6 1 22 23 4
9 14 21 20 23 18 5 16 28 10
6 3 6 22 28 20 19
3 3 5 27
6 30 16 19 28 13 20
5 6 15 28 11 27
95 13
9 13 10 4 14 6 6 19 19 25
12 7 30 2 12 18 5 3 25 24 30 30 5
7 5 11 16 7 29 24 1
5 6 11 16 20 8
11 30 26 30 30 2 29 23 5 8 20 7
9 5 29 24 1 17 20 3 24 26
5 17 18 14 14 28
9 3 27 26 3 22 25 24 16 15
10 12 14 27 19 8 1 17 23 23 25
14 25 18 7 26 26 25 1 9 14 13 12 11 30 6
8 24 21 1 30 28 13 6 17
9 5 9 9 28 2 25 9 26 4
4 13 29 24 13
59 19
12 10 1 17 7 25 3 19 25 2 17 7 29
5 29 4 11 29 1
12 23 1 22 18 8 4 16 1 9 15 22 26
8 22 12 23 8 14 11 3 8
7 1 6 21 22 1 24 20
3 19 5 24
6 22 23 6 30 16 6
12 7 1 29 28 4 21 6 10 2 30 17 20
6 14 3 13 7 18 25
7 6 21 22 8 4 15 5
13 30 10 16 6 2 15 25 5 27 22 14 20 21
8 2 13 6 4 25 4 13 19
14 10 9 24 9 13 30 13 7 21 14 23 18 15 29
6 12 17 26 25 7 16
13 8 20 6 3 15 9 15 25 10 16 26 25 25
10 16 29 6 6 13 20 23 19 10 27
14 27 14 17 3 29 25 2 19 30 4 25 30 10 11
3 25 6 26
11 5 11 10 3 17 14 22 1 2 2 19
74 8
12 18 22 23 4 23 3 3 26 19 24 27 29
4 13 5 27 24
1 30
3 3 8 8
13 8 9 26 19 2 15 12 11 6 26 14 28 28
6 15 8 1 3 7 3
5 11 21 8 11 20
4 14 27 23 30
74 2
1 7
3 1 10 6
43 16
11 17 25 27 23 3 27 25 9 21 9 11
2 9 13
9 2 26 4 17 17 30 18 3 28
6 25 7 9 14 4 20
6 21 8 22 23 26 8
13 17 17 3 12 17 15 26 10 3 29 26 19 20
1 14
13 9 30 16 9 14 11 28 13 1 5 27 23 1
10 7 9 12 1 12 28 16 7 30 10
4 25 20 9 30
3 18 30 3
3 8 16 5
14 20 6 2 8 20 24 4 19 2 7 19 13 27 4
14 26 5 1 12 25 9 11 27 18 10 21 12 10 28
9 7 10 6 8 9 17 2 4 5
11 11 24 30 29 27 3 16 2 3 27 18
93 10
12 21 1 27 24 10 16 24 16 25 29 16 4
10 9 7 20 3 9 13 24 7 2 26
10 25 20 11 12 23 10 17 5 10 13
14 20 29 14 27 15 4 12 10 19 20 9 1 15 17
5 8 16 6 26 29
10 15 1 11 29 10 20 25 12 2 15
9 22 20 19 29 23 23 8 4 12
6 4 18 25 8 26 2
14 21 22 5 27 23 16 26 2 5 20 5 28 4 27
4 15 16 10 30
55 7
3 11 25 6
3 19 5 16
2 10 6
3 15 24 3
14 19 27 26 9 1 15 4 28 26 11 5 27 10 4
8 12 14 30 9 4 18 13 19
3 23 16 9
87 10
2 20 20
8 7 28 22 21 24 11 9 4
3 5 13 10
4 16 16 9 24
11 18 28 30 16 12 7 25 10 8 20 29
4 19 5 25 2
8 10 5 26 13 11 22 17 21
4 2 28 24 25
12 12 15 7 19 26 6 13 6 13 2 4 10
2 9 27
40 16
14 10 13 11 20 4 27 2 19 21 29 13 7 7 16
5 13 4 17 10 9
8 23 2 9 2 6 9 28 11
4 26 20 8 6
11 4 2 3 22 22 1 26 21 7 3 11
8 29 19 21 7 2 5 30 11
11 5 19 26 15 15 21 26 14 26 19 17
8 21 1 3 13 26 23 12 21
8 23 19 14 5 17 16 10 16
6 8 21 28 3 27 4
8 15 18 11 3 4 22 23 26
11 27 22 9 8 4 4 30 22 18 27 8
5 28 23 4 5 5
3 29 2 5
14 16 14 17 10 9 8 24 5 24 20 18 25 28 21
12 19 12 15 15 19 1 4 3 4 30 8 27
56 11
3 26 8 6
12 17 14 12 10 10 27 29 27 21 18 17 19
5 20 25 13 30 18
5 3 21 8 2 17
12 2 9 24 9 14 27 25 20 30 26 29 27
14 18 9 4 26 19 10 16 14 14 15 1 30 9 13
6 10 30 27 11 8 20
5 14 17 27 25 16
6 15 4 30 2 13 3
14 1 4 5 6 17 11 28 9 20 11 16 29 10 4
6 9 24 4 14 2 1
64 19
8 7 13 13 8 17 15 19 9
8 23 15 27 4 12 27 15 14
14 5 23 7 6 24 30 2 29 23 2 21 2 15 27
6 28 26 22 4 15 22
6 7 6 2 2 10 28
4 23 1 13 8
11 10 1 21 11 21 13 12 11 6 27 7
9 16 25 24 11 9 16 16 7 13
3 9 22 28
12 15 20 28 22 19 8 14 9 18 26 21 22
11 19 10 27 29 25 21 15 27 29 22 13
12 4 13 6 18 11 21 2 30 11 15 10 10
10 18 19 15 1 10 13 19 11 2 9
13 22 15 24 21 6 6 18 2 18 23 19 20 5
2 12 7
8 21 16 15 1 27 21 23 6
14 11 9 27 11 13 18 26 28 30 23 3 9 24 13
3 12 24 28
7 5 4 18 26 12 24 26
96 15
6 5 10 20 13 28 22
4 15 9 15 7
2 10 15
9 14 8 29 7 27 21 12 22 1
3 3 24 24
4 30 3 7 1
12 11 28 5 20 5 14 5 3 7 6 17 23
5 16 21 17 12 4
10 4 4 18 28 19 3 30 11 5 28
5 10 30 1 15 20
13 20 16 7 18 21 15 10 1 1 1 9 4 4
13 29 29 16 27 17 19 18 19 15 7 30 25 28
12 1 17 26 12 2 24 29 14 30 8 6 22
4 15 26 25 13
12 23 20 12 2 8 21 20 15 27 11 1 17
60 11
7 27 4 27 12 2 2 12
13 8 3 1 14 20 17 18 6 10 7 17 11 7
6 22 21 26 3 13 12
14 13 7 23 8 25 4 1 26 7 1 25 2 23 8
7 10 25 18 11 24 5 13
10 4 4 12 21 6 24 25 2 28 1
4 5 17 19 5
8 18 27 29 19 12 28 1 13
7 19 23 8 15 5 7 18
2 10 30
2 3 24
48 2
6 22 26 2 2 22 6
13 10 4 29 21 1 30 3 15 18 25 22 24 29
41 11
11 29 10 26 23 26 24 14 11 7 1 5
8 22 10 27 2 13 25 14 5
6 17 20 25 3 3 18
8 23 20 22 13 30 9 6 17
4 11 19 8 12
7 15 25 24 3 18 28 20
6 3 5 10 14 30 12
14 9 27 8 29 18 13 20 18 18 6 19 20 24 26
5 9 3 18 2 27
7 22 16 28 24 13 29 7
12 3 22 12 29 22 10 9 4 29 26 13 26
67 2
2 2 25
8 26 12 13 15 8 4 30 28
43 14
6 25 7 28 9 19 27
12 20 5 17 19 22 29 6 28 30 17 22 16
4 17 27 11 1
7 6 23 2 25 26 19 11
13 17 11 5 5 24 24 1 10 4 22 8 10 20
12 26 11 15 24 19 4 26 12 30 23 4 23
1 21
5 12 15 19 14 19
3 8 5 23
7 8 7 17 9 26 8 27
14 14 20 8 9 7 19 8 29 14 30 30 27 3 11
5 21 24 14 6 23
2 28 10
9 4 18 26 21 17 22 10 30 3
57 11
7 20 8 21 4 30 20 22
6 22 16 14 16 29 19
2 8 9
13 25 4 18 12 17 4 25 18 26 19 27 26 19
5 25 9 12 16 29
7 9 12 10 15 27 9 3
9 16 3 27 2 29 15 13 15 10
5 2 27 17 21 14
3 7 9 14
2 16 4
8 25 15 13 9 4 21 3 22
65 8
6 30 26 24 5 10 26
11 3 22 12 23 28 16 21 6 21 30 13
6 3 7 23 8 7 26
4 10 9 18 29
4 10 24 12 6
1 29
14 27 20 11 12 9 27 2 14 17 24 19 25 18 25
7 25 2 4 15 3 12 24
92 19
9 16 2 8 10 30 30 28 12 10
13 20 6 3 26 15 26 14 9 13 30 25 30 1
4 14 25 1 7
13 29 2 24 23 1 3 22 30 1 25 10 1 15
5 3 2 21 28 15
1 3
12 7 2 7 26 15 23 26 13 2 25 6 25
3 29 28 30
14 20 25 29 20 1 28 23 2 10 12 8 1 14 13
1 15
3 3 21 4
14 26 5 6 23 29 14 21 18 14 11 29 30 1 19
8 28 3 23 8 14 22 30 28
8 28 4 15 22 25 18 12 12
4 17 4 12 1
1 22
12 27 20 5 27 8 18 25 10 10 24 16 23
3 13 26 20
8 10 12 2 19 15 5 2 1
62 16
1 25
9 6 21 8 2 9 16 19 3 17
8 26 24 4 10 6 30 22 14
13 25 7 20 9 12 13 9 12 18 2 28 14 29
12 22 23 18 29 3 21 7 15 8 1 18 10
12 9 1 3 10 25 10 21 25 13 25 25 16
9 26 13 18 17 22 1 9 9 29
5 21 28 17 21 20
11 30 17 13 30 12 14 16 21 26 10 25
11 4 10 3 22 14 12 8 27 13 8 5
9 10 18 30 19 8 11 23 29 20
3 28 1 3
13 13 20 14 7 10 9 8 4 22 21 16 29 9
14 28 13 22 30 22 22 18 21 24 2 19 13 20 8
3 22 12 17
6 17 15 13 18 22 17
52 14
6 8 12 13 27 17 4
10 30 17 5 21 11 6 9 15 26 17
2 9 20
2 12 7
1 25
10 26 3 24 30 26 1 3 30 20 11
3 15 11 20
8 1 22 17 1 28 12 9 17
7 29 15 24 27 19 10 12
2 12 5
3 29 28 8
2 17 18
5 23 28 12 4 20
7 21 13 22 24 21 9 6
100 15
4 29 3 1 10
12 5 7 21 3 26 20 1 12 7 23 26 27
4 30 8 28 20
8 19 5 3 19 3 13 25 24
2 27 16
3 5 20 11
3 23 6 6
1 9
14 15 5 30 10 26 30 29 15 11 9 11 13 28 13
2 14 6
1 11
10 3 7 11 6 23 25 3 28 9 12
13 23 8 23 24 3 14 22 9 24 23 19 7 12
10 24 25 29 14 27 20 8 3 23 5
7 17 8 14 26 19 7 10
