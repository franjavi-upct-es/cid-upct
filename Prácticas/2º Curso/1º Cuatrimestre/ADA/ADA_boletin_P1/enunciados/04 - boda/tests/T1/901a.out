90
85
68
no solution
78
40
77
no solution
50
no solution
no solution
82
no solution
71
no solution
43
54
98
93
no solution
82
no solution
49
45
no solution
28
97
95
no solution
74
17
no solution
93
55
87
no solution
no solution
no solution
96
60
48
no solution
55
no solution
no solution
65
no solution
no solution
no solution
100
